nomad/local/grapl-local-infra.nomad

```diff
@@ -3,24 +3,60 @@ variable "image_tag" {
   description = "The tag for all container images we should deploy. This is ultimately set in the top-level Makefile."
 }
 
-variable "kafka_broker_port" {
+variable "kafka0_broker_port" {
```

Comment:
I think I'd much prefer a 
```
variable "kafka_broker_args" {
  type = list(object({
    broker_port_listen = number
    broker_port_host_os = number
    jmx_port = number
  }))
}
```

but I won't push the matter;  notably this would require you constructing that arg as json in startdetach

Reply:
Oh awesome, thanks! I'll do that and use a dynamic task.

Reply:
done